modules/manage/pages/cluster-maintenance/disk-utilization.adoc

```diff
@@ -88,17 +88,18 @@ See also:
 
 === Set time-based retention
 
-Messages are eligible for deletion if their age exceeds the value specified in `delete_retention_ms` (the cluster-level property) or `retention.ms` (the topic-level property). If `retention.ms` is not set at the topic level, the topic inherits the `delete_retention_ms` setting. Deletion occurs at a segment level and open segments are never deleted. Only closed segments where all messages in the segment exceed the specified retention time are deleted.
+Messages are eligible for deletion once their age exceeds the value specified in `delete_retention_ms` (the cluster-level property) or `retention.ms` (the topic-level property). Only closed segments are eligible for deletion and all messages in a closed segment must exceed the age limit before Redpanda considers the segment for cleanup. If `retention.ms` is not set at the topic level, the topic inherits the `delete_retention_ms` setting.
```

Comment:
Issue here....`delete_retention_ms` is deprecated in this release...not sure how this passes muster with eng.
@andijcr ?
This was implemented in https://github.com/redpanda-data/docs/pull/161

You should tell them to use xref:upgrade:deprecated/cluster-resource.adoc[log_retention_ms] instead

Reply:
Right, should change from delete_retention_ms to log_retention_ms (inside redpanda it's just a rename)

Reply:
We had a bunch of references to the old field, I've fixed them all to be `log_retention_ms`. I thought we'd fixed this under your PR, @Feediver1 , sorry if a merge somewhere clobbered it!